packages/babel-plugin-redux-saga/README.md

```diff
@@ -46,23 +46,14 @@ saga2[_SAGA_LOCATION] = {
 };
 ```
 
-## Install
-add to package.json
-```json
-    "devDependencies": {
-        ...
-        "babel-plugin-transform-redux-saga-source": "github:restrry/babel-plugin-transform-redux-saga-source"
-    }
-```
-
 ## Usage
 
 1. with babel
 ```js
 babel.transform(content, {
     sourceMaps: true,
     plugins: [
-        'babel-plugin-transform-redux-saga-source', { /* options */ }
+        'babel-plugin-redux-saga', { /* options */ }
```

Comment:
we have a typo here, should be `['babel-plugin-redux-saga', { /* options */ }]` or mention of the options should get removed from here